### DISCUSSION TITLE
Welcome to reactjs-social-login Discussions!

### DISCUSSION THREAD

**Discussion by cuongdevjs** [cuongdevjs/reactjs-social-login#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cuongdevjs** [comment]
If you have any questions or any ideas or want to add any provider OAuth to add to this project. Feel free to tell me know :)

**Comment by ArunJose-WTC** [comment]
there is no option to get user's email ID with LoginSocialFacebook, Can you update the fieldsProfile with email?

**Comment by cuongdevjs** [comment]
Sure. Currently, I just list some props on README, you can read more props
on the link of each part on README.
In your case, you can pass fieldsProfile on LoginSocialFacebook, more
detail:
https://github.com/cuongdevjs/reactjs-social-login/blob/333e6914bbc50b81e94137a5f7e12d3b60ba04b2/src/LoginSocialFacebook/index.tsx?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L33

Let me know if you have any questions.
Thanks.
Regards

**Reply by ArunJose-WTC** [reply]
> Sure. Currently, I just list some props on README, you can read more props on the link of each part on README. In your case, you can pass fieldsProfile on LoginSocialFacebook, more detail: https://github.com/cuongdevjs/reactjs-social-login/blob/333e6914bbc50b81e94137a5f7e12d3b60ba04b2/src/LoginSocialFacebook/index.tsx?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L33 Let me know if you have any questions. Thanks. Regards
> […](#)
> On Fri, Dec 31, 2021 at 12:04 PM ArunJose-WTC ***@***.***> wrote: there is no option to get user's email ID with LoginSocialFacebook, Can you update the fieldsProfile with email? — Reply to this email directly, view it on GitHub <[#23 (comment)](https://github.com/cuongdevjs/reactjs-social-login/discussions/23#discussioncomment-1890368)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AIAMGXSNBNU66WL3FQEOYULUTU2UDANCNFSM5FB4YHFQ> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. You are receiving this because you authored the thread.Message ID: <cuongdevjs/reactjs-social-login/repo-discussions/23/comments/1890368@ github.com>

Hi,
I have tried that.
fieldsProfile={'id,email,first_name,last_name,middle_name'}

still not getting email.

**Reply by cuongdevjs** [reply]
Can you give me an example? I just pass props fieldsProfile on my example and it works perfectly. You can check it on: https://react-social-login.netlify.app

This is my code
````
                               <LoginSocialFacebook
					appId={"APP_ID"}
					onLoginStart={onLoginStart}
					fieldsProfile={
						"id,first_name,last_name,middle_name,name,name_format,picture,short_name,email"
					}
					onLogoutSuccess={onLogoutSuccess}
					onResolve={({ provider, data }) => {
						setProvider(provider);
						setProfile(data);
					}}
					onReject={(err) => {
						console.log(err);
					}}
				>
					<FacebookLoginButton />
				</LoginSocialFacebook>
````

**Reply by ArunJose-WTC** [reply]
Thank you :)
